assets/styles/_base.scss

```diff
@@ -135,6 +135,45 @@ body {
   }
 }
 
+.single-post {
+  .post {
+    &-header + .post-main {
+      margin: 1.375rem 0 0;
+
+      @media (min-width: $breakpoint-tablet) {
+        margin-top: 2rem;
+      }
+    }
+
+    &-main + .post-footer {
+      margin: 2.75rem 0 0;
+
+      @media (min-width: $breakpoint-tablet) {
+        margin-top: 3rem;
+      }
+    }
+
+    &-main,
+    &-related {
+      > .flyntComponent {
+        margin: 0;
```

Comment:
This shouldn't be the case. At most the first and last components margin-/padding-top / -bottom.